src/components/extractionPipelinesTab.tsx

```diff
@@ -32,8 +31,8 @@ export const ExtractionPipelinesTab = (
   return (
     <div style={{ marginTop: 8 }}>
       <div className="gf-form-inline">
-        <Field label="Collumns">
-          <Tooltip content="Add or remove columns">
+        <Field label="Columns" description="Add or remove columns">
```

Comment:
does the description render as a tooltip?

Reply:
tool tip was weird (showed up in the middle of the row), so I used description instead. you can see it on the screenshots above